src/WorkspaceDependencyViewExtension/WorkspaceDependencyView.xaml.cs

```diff
@@ -116,6 +116,14 @@ internal void DependencyRegen(WorkspaceModel ws)
             PackageDependencyTable.ItemsSource = packageDependencies.Select(d => new PackageDependencyRow(d as PackageDependencyInfo));
         }
 
+        /// <summary>
+        /// Calls the DependencyRegen function when the event is triggered from the dynamo model.
+        /// </summary>
+        internal void TriggerDepenRegen()
```

Comment:
whats going on here? Why does this need to live in the view?

Reply:
This is just a helper method to call the DependencyRegen function.